test/integration/integration_test.go

```diff
@@ -455,6 +477,7 @@ func executingOcCommandSucceedsOrFails(command string, expectedResult string) er
 }
 
 func executingMinishiftCommandSucceedsOrFails(command string, expectedResult string) error {
+	fmt.Println(command)
```

Comment:
@hferentschik Forgotten line from debugging the problem with executing command step, will be removed.